src/content/en/updates/2019/08/contact-picker.md

```diff
@@ -144,7 +146,7 @@ The Contacts Picker API can only be called from a [secure][secure-contexts],
 top-level browsing context, and like other powerful APIs, it requires a
 user gesture.
 
-#### Handling the results
+### Handling the results
```

Comment:
I promoted this heading because:

* It doesn't seem less important than other level 3s. 
* It doesn't show up in the TOC.